docs/components/drawer/slotted-drawer-demo.js

```diff
@@ -0,0 +1,64 @@
+import Util from '../../_util';
+
+if (document.getElementById('vue-slottedDrawerDemo')) {
+    new Vue({
+        el: '#vue-slottedDrawerDemo',
+        data: {
+            hasSlottedPadding: true,
+            hasSlottedLengthyContent: false,
+            drawerTitle: 'Drawer with Slotted Positionable Element',
+        },
+        computed: {
+            attrBodyClass: function () {
+                if (this.bodyClasses !== '') {
+                    return `class="${this.bodyClasses}"`;
+                }
+                return '';
+            },
+            bodyClasses: function () {
+                return this.hasSlottedPadding ? 'hxMd' : '';
+            },  
+            snippet: function () {
+                return Util.snippet(`
+                    <hx-disclosure
+                      aria-controls="drawer-with-slotted-positionable-element"
+                      class="hxBtn"
+                    >
+                    Drawer with Slotted Positionable Element
+                    </hx-disclosure>
+
+                    <hx-drawer
+                      id="drawer-with-slotted-positionable-element"
+                    >
+                      <header></header>
+
+                      <hx-div ${this.attrBodyClass}>
+                        ...
+                      </hx-div>
+
+                      <footer>
+                        ...
+                      </footer>
```

Comment:
Please move this below the positionable elements.

```html
  <footer>
    . . .
  </footer>
</hx-drawer>
```

Reply:
Changes have been done, please review.